### DISCUSSION TITLE
How is checksum computed?

### DISCUSSION THREAD

**Discussion by DennisHeimbigner** [OPENDAP/dap4-specification#6] [General]
I have discovered that netcdf and opendap compute the DAP checksum for a variable in different ways.
In netcdf, I take the part of the DAP4 data stream corresponding to a variable and compute the
checksum over that "sub-chunk".
In Opendap (Hyrax), the code picks out the data only and computes the checksum over that.

The difference then is that the netcdf checksum includes the counts for variable length data
and Hyrax excludes the counts.

I suspect that the netcdf approach is faster, but the difference is probably very small.

**Comment by DennisHeimbigner** [comment]
You are correct. The issue is under discussion here: https://github.com/OPENDAP/dap4-specification/discussions/6

**Comment by jhrg** [comment]
Dennis, I'm late to the discussion. What are the 'counts?'
